cmd/cosign/cli/options/signblob.go

```diff
@@ -71,6 +72,9 @@ func (o *SignBlobOptions) AddFlags(cmd *cobra.Command) {
 		"write everything required to verify the blob to a FILE")
 	_ = cmd.Flags().SetAnnotation("bundle", cobra.BashCompFilenameExt, []string{})
 
+	cmd.Flags().BoolVar(&o.UsePBBundleFormat, "use-new-bundle-format", false,
```

Comment:
Can we add a TODO to eventually make this flag a no-op and add a deprecated-format flag?